From e240a560033e90f3ac5d76ab77daa2360f37c8c5 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Fri, 13 Sep 2024 12:10:29 +0200
Subject: [PATCH 1151/1695] dts: stm32: let ETR to access kernel memory on
 stm32mp215f-dk board

Configure rifsc for ETR

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 2986bcc6d75e59333f2e4a40a830fc41531fc308 dts: stm32: let ETR to access kernel memory on stm32mp215f-dk board]
Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Change-Id: I55a12b343df54b4f5bec9976b2e4bcb408b861b3
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/404325
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rif.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rif.dtsi b/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rif.dtsi
index bea329481..9945aeb18 100644
--- a/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rif.dtsi
+++ b/core/arch/arm/dts/stm32mp215f-dk-ca35tdcid-rif.dtsi
@@ -537,7 +537,7 @@
 
 &rifsc {
 	st,rimu = <
-		RIMUPROT(RIMU_ID(0), RIF_UNUSED, RIF_NSEC, RIF_PRIV, RIF_CIDSEL_P)
+		RIMUPROT(RIMU_ID(0), RIF_CID1, RIF_NSEC, RIF_PRIV, RIF_CIDSEL_M)
 		RIMUPROT(RIMU_ID(1), RIF_UNUSED, RIF_NSEC, RIF_PRIV, RIF_CIDSEL_P)
 		RIMUPROT(RIMU_ID(2), RIF_UNUSED, RIF_NSEC, RIF_NPRIV, RIF_CIDSEL_P)
 		RIMUPROT(RIMU_ID(3), RIF_UNUSED, RIF_NSEC, RIF_PRIV, RIF_CIDSEL_P)
-- 
2.47.2

